tgintegration/containers/response.py

```diff
@@ -108,12 +101,12 @@ def last_message_timestamp(self) -> Optional[datetime]:
         if self.empty:
             return None
         # TODO: Dan should fix this
```

Comment:
Who is Dan?

Reply:
https://t.me/haskell <-- Pyrogram author

Reply:
Ah, okay